tests/page-objects/forms/cht/delivery.po.js

```diff
@@ -0,0 +1,140 @@
+const helper = require('../../../helper');
+const genericForm = require('../generic-form.po');
+const moment = require('moment');
+
+const inputBase = 'input[name="/delivery/';
+
+const condition = `${inputBase}condition/woman_outcome"]`;
```

Comment:
I'm considering a function that you pass the css selector in and the value but I'm not sure if it is any better than this.  I think it might be a little cleaner as you could pull some away but then your css selectors show as half built with variables. Interested in thoughts?

Something like 

```
function selectValue(cssSelector, value) {
   element(by.css(`${cssSelector}[value=${value}]`));
}
```

and calling would be like 

```
po.selectValue('input[name="/delivery/pnc_danger_sign_check/fever"', 'no']
```